session/session.go

```diff
@@ -2359,8 +2403,18 @@ func runStmt(ctx context.Context, se *session, s sqlexec.Statement) (rs sqlexec.
 				}
 			}
 		}
+
+		copiedRs := rs
```

Comment:
May need a switch to turn it on or off the hook.

Reply:
I've set the maximum number of rows to store and log to 5. Do you think it still necessary to add a switch?

Reply:
Fine with me, the limit `5` is a little bit small, maybe `50`. If it's enough to cover the questionable query that's OK.